Userland/Libraries/LibCore/File.h

```diff
@@ -55,9 +55,11 @@ class File final : public IODevice {
         No,
     };
 
-    enum class PreserveMode {
-        Nothing,
-        PermissionsOwnershipTimestamps,
+    enum PreserveMode {
```

Comment:
Plain `enum`? :catdogalert: Pretty sure you want to use `AK_ENUM_BITWISE_OPERATORS` instead.

Reply:
That's what I was looking for but could not find. 🙃 

The code looks much better now, and the commits have the correct category. :^)